proto/info/info.proto

```diff
@@ -495,21 +496,13 @@ message SmartAttr {
   uint64 thresh = 4;  // the value that the Value of the same attribute must reach for the attribute's state to be considered critical.
   string when_failed = 5; // critical state if no empty
   uint64 raw_value = 6; // Contains miscellaneous meter readings (where applicable)
-}
-
-message SmartMetric {
```

Comment:
Its fine to remove this, but can you check (e.g., with grep) that neither Adam nor Eden refers to this message.